presto/client.py

```diff
@@ -551,3 +563,7 @@ def cancel(self):
     def is_finished(self):
         # type: () -> bool
         return self._finished
+
+    @property
+    def response_headers(self):
```

Comment:
That does not apply to `post`, `get`, and `delete` because they belong to the `PrestoRequest` instead of the `PrestoQuery` class. Did you mean `execute` and `cancel` instead?

Reply:
You are correct. I should've said `execute` and `cancel`. I checked the headers returned on `execute` and `cancel` but they didn't look useful enough generally (yet) so we can let this go.